pkg/fscrypt/build.yml

```diff
@@ -0,0 +1,8 @@
+# linuxkit build template 
+#
+# SPDX-License-Identifier: Apache-2.0
+org: lfedge
+image: eve-fscrypt
+# we allow network in building fscrypt because we need it to downlaod go modules,
+# and it is limited to just this single target output.
+network: yes
```

Comment:
Why do you need network? You get the code using ADD

Reply:
It isn't enough, which is why I moved it into its own `pkg/fscrypt/`. When we run `go build`, it downloads all of the dependencies. There is no way around that for now, or I would have left it in pillar.

Reply:
Right - thanks for the reminder.